src/main/java/net/snowflake/client/core/SFSession.java

```diff
@@ -486,6 +486,12 @@ public void addSFSessionProperty(String propertyName, Object propertyValue) thro
           }
           break;
 
+        case JDBC_DEFAULT_FORMAT_DATE_WITH_TIMEZONE:
+          if (propertyValue != null) {
+            setUseHardcodedTimezone(getBooleanValue(propertyValue));
```

Comment:
please rename also internal naming (field, setters and getters) of the parameter to be the same as property name